### PR TITLE
Add blog subfolder 3

### PR DESCRIPTION
**Pull request by naterush**
# Description

Cha.

# Testing

Cha.

# Documentation

Note if any new documentation needs to addressed or reviewed.

### CODE REVIEW

File: trymito.io/components/Header/Header.tsx [hunk 1]
```diff
@@ -75,7 +75,7 @@ const Header = (): JSX.Element => {
                   <Link href='/security'>Security</Link>
                 </li>
                 <li className={classNames('highlight-on-hover', headerStyles.nav_item)}>
-                  <Link href='https://blog.trymito.io'>Blog</Link>
```
Comment (naterush) [inline trymito.io/components/Header/Header.tsx:78]: Blog

File: trymito.io/pages/spreadsheet-automation.tsx [hunk 2]
```diff
@@ -74,7 +74,7 @@ const SpreadsheetAutomation: NextPage = () => {
                   Edit your data just like you do in Excel, and receive production ready Python code. No searching Pandas documentation or Stack Overflow.
                 </p>
                 <p>
-                  See our <a href="https://blog.trymito.io/automating-spreadsheets-with-python-101/" target="_blank" rel="noreferrer" className={pageStyles.link}>guide to automating spreadsheet with Python.</a>
+                  See our <a href="https://trymito.io/blog/automating-spreadsheets-with-python-101/" target="_blank" rel="noreferrer" className={pageStyles.link}>guide to automating spreadsheet with Python.</a>
```
Comment (naterush) [inline trymito.io/pages/spreadsheet-automation.tsx:77]: The should be <Link>

**Review by naterush — COMMENTED** [review]
Cleanup

**Comment by aarondr77** [comment]
This first blog doesn't look like a blog post to me. I thought it was the title of our blog! Would be down to not separate this blog and just have it in the list

Screenshot 2023-09-06 at 12 49 14 PM

**Comment by aarondr77** [comment]
We should add a bit more space between the metadata and the title 

Screenshot 2023-09-06 at 12 51 53 PM

**Comment by aarondr77** [comment]
Picture captions should be smaller, lighter, and center aligned 

Screenshot 2023-09-06 at 12 54 16 PM

**Comment by aarondr77** [comment]
The Blog Title should be larger than the H1 subsection titles 

Screenshot 2023-09-06 at 12 56 01 PM

**Comment by aarondr77** [comment]
I think the CTA at the bottom should just be the download CTA instead of the contact one?

**Comment by aarondr77** [comment]
This is so text-dense. Some ideas:
1. Reduce the amount of preview text. 
2. Make the blog cards not take up the entire horizontal space. Maybe leave some space on either side. 

Screenshot 2023-09-06 at 12 58 18 PM

**Comment by aarondr77** [comment]
Also, btw this is branched off an old version of the website that is missing a bunch of foundational pages. I bet there are other blog links to update too

**Pull request merged**